### ISSUE TITLE
update README setup instructions

### ISSUE THREAD

**Issue by bbelderbos** [PyBites-Open-Source/pybites-carbon#21]
On 2nd system I got this today:

```
playwright._impl._errors.Error: BrowserType.launch: Executable doesn't exist at ....
╔════════════════════════════════════════════════════════════╗
║ Looks like Playwright was just installed or updated.       ║
║ Please run the following command to download new browsers: ║
║                                                            ║
║     playwright install                                     ║
║                                                            ║
║ <3 Playwright Team                                         ║
╚════════════════════════════════════════════════════════════╝
```

So I think we need to add this to the README, no? `uv tool run playwright install`

@cmsato09

**Comment by cmsato09** [comment]
I had the same problem when I was fixing the workflow file. 
`uv run playwright install chromium` step is needed in the README.
`install` by itself installs the default browser, but we did write it with chromium so better to be explicit(?)

https://playwright.dev/python/docs/browsers#install-browsers

**Comment by bbelderbos** [comment]
ok cool, do you want to do that + review the readme? If busy, I can do it too ...

**Comment by cmsato09** [comment]
> ok cool, do you want to do that + review the readme? If busy, I can do it too ... - thanks

I got it. I'm working on a draft.

**Comment by bbelderbos** [comment]
Thanks Chris, published 0.7.2 with this update readme.
